test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1699,7 +1710,7 @@ describe('setAudioMute()', () => {
   });
 
   it('should throw if there are errors', async () => {
-    const params: ISessionMuteRequest = { conversationId: null, mute: true };
+    const params: ISessionMuteRequest = { conversationId: null as any, mute: true };
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    const params: ISessionMuteRequest = { conversationId: null as unknown, mute: true };
```